<?xml version="1.0" encoding="UTF-8"?>
<TestRun id="ba52a71d-8d6c-41c4-9fdc-5011ce6f56f7" name="Truong Tho@TRUONGTHO-PC 2012-05-17 23:42:27" runUser="TruongTho-PC\Truong Tho" xmlns="http://microsoft.com/schemas/VisualStudio/TeamTest/2010">
  <TestSettings name="Local" id="0210f5e3-9eec-4396-b5fb-9dc61770bc14">
    <Description>These are default test settings for a local test run.</Description>
    <Deployment enabled="false" runDeploymentRoot="Truong Tho_TRUONGTHO-PC 2012-05-17 23_42_27">
      <DeploymentItem filename="F:\Workplace\WPF-WCF\SourceProject\kyan-projects\DemoRalcon\DemoRalcon\bin\Debug\DemoRalcon.exe" />
      <DeploymentItem filename="C:\Program Files (x86)\System.Data.SQLite\2010\bin\System.Data.SQLite.dll" />
    </Deployment>
    <Execution>
      <TestTypeSpecific />
      <AgentRule name="Execution Agents">
      </AgentRule>
    </Execution>
  </TestSettings>
  <Times creation="2012-05-17T23:42:27.9669217+07:00" queuing="2012-05-17T23:42:28.1279309+07:00" start="2012-05-17T23:42:28.1399316+07:00" finish="2012-05-17T23:42:28.5189533+07:00" />
  <ResultSummary outcome="Failed">
    <Counters total="1" executed="1" error="0" failed="1" timeout="0" aborted="0" inconclusive="0" passedButRunAborted="0" notRunnable="0" notExecuted="0" disconnected="0" warning="0" passed="0" completed="0" inProgress="0" pending="0" />
  </ResultSummary>
  <TestDefinitions>
    <UnitTest name="InsertTest" storage="f:\workplace\wpf-wcf\sourceproject\kyan-projects\demoralcon\dataaccesstest\bin\debug\dataaccesstest.dll" id="f4542e4d-839a-7b11-2a53-610b5e150678">
      <Execution id="2f7eb39d-539d-41c3-9c6c-591f05136c2f" />
      <TestMethod codeBase="F:/Workplace/WPF-WCF/SourceProject/kyan-projects/DemoRalcon/DataAccessTest/bin/Debug/DataAccessTest.DLL" adapterTypeName="Microsoft.VisualStudio.TestTools.TestTypes.Unit.UnitTestAdapter, Microsoft.VisualStudio.QualityTools.Tips.UnitTest.Adapter, Version=10.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a" className="DataAccessTest.DepartmentDetailDataAccessTest, DataAccessTest, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null" name="InsertTest" />
    </UnitTest>
  </TestDefinitions>
  <TestLists>
    <TestList name="Results Not in a List" id="8c84fa94-04c1-424b-9868-57a2d4851a1d" />
    <TestList name="All Loaded Results" id="19431567-8539-422a-85d7-44ee4e166bda" />
  </TestLists>
  <TestEntries>
    <TestEntry testId="f4542e4d-839a-7b11-2a53-610b5e150678" executionId="2f7eb39d-539d-41c3-9c6c-591f05136c2f" testListId="8c84fa94-04c1-424b-9868-57a2d4851a1d" />
  </TestEntries>
  <Results>
    <UnitTestResult executionId="2f7eb39d-539d-41c3-9c6c-591f05136c2f" testId="f4542e4d-839a-7b11-2a53-610b5e150678" testName="InsertTest" computerName="TRUONGTHO-PC" duration="00:00:00.0900368" startTime="2012-05-17T23:42:28.1529324+07:00" endTime="2012-05-17T23:42:28.5019523+07:00" testType="13cdc9d9-ddb5-4fa4-a97d-d965ccfc6d4b" outcome="Failed" testListId="8c84fa94-04c1-424b-9868-57a2d4851a1d" relativeResultsDirectory="2f7eb39d-539d-41c3-9c6c-591f05136c2f">
      <Output>
        <DebugTrace>
|| === Exception === 
|| Source :DemoRalcon
|| Message :Object reference not set to an instance of an object.
|| StackTrace :   at DemoFalcon.DataAccess.DepartmentDetailDataAccess.Insert(DepartmentDetailModel departmentDetailModel) in F:\Workplace\WPF-WCF\SourceProject\kyan-projects\DemoRalcon\DemoRalcon\DataAccess\DepartmentDetailDataAccess.cs:line 255
|| TargetSite :Boolean Insert(DemoFalcon.Model.DepartmentDetailModel)
|| Data :
|| All :System.NullReferenceException: Object reference not set to an instance of an object.
||   at DemoFalcon.DataAccess.DepartmentDetailDataAccess.Insert(DepartmentDetailModel departmentDetailModel) in F:\Workplace\WPF-WCF\SourceProject\kyan-projects\DemoRalcon\DemoRalcon\DataAccess\DepartmentDetailDataAccess.cs:line 255
|| Data System.Collections.ListDictionaryInternal
</DebugTrace>
        <ErrorInfo>
          <Message>Test method DataAccessTest.DepartmentDetailDataAccessTest.InsertTest threw exception: 
System.NullReferenceException: Object reference not set to an instance of an object.</Message>
          <StackTrace>    at DemoFalcon.DataAccess.DepartmentDetailDataAccess.Insert(DepartmentDetailModel departmentDetailModel) in F:\Workplace\WPF-WCF\SourceProject\kyan-projects\DemoRalcon\DemoRalcon\DataAccess\DepartmentDetailDataAccess.cs:line 270
   at DataAccessTest.DepartmentDetailDataAccessTest.InsertTest() in F:\Workplace\WPF-WCF\SourceProject\kyan-projects\DemoRalcon\DataAccessTest\DepartmentDetailDataAccessTest.cs:line 150
</StackTrace>
        </ErrorInfo>
      </Output>
    </UnitTestResult>
  </Results>
</TestRun>